ecosystem/web-wallet/src/core/components/NetworkListItem.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  Box,
+  Center,
+  Spinner,
+  Text,
+  useColorMode,
+  useRadio,
+  UseRadioProps,
+} from '@chakra-ui/react';
+import { LOCAL_NODE_URL } from 'core/constants';
+import {
+  AptosNetwork,
+  networkUriMap,
+} from 'core/utils/network';
+import React from 'react';
+
+export interface SettingsListItemProps {
+  title?: 'Mainnet' | 'Testnet' | 'Devnet' | 'Localhost';
+  value: AptosNetwork;
+}
+
+export const secondaryBgColor = {
+  dark: 'gray.600',
+  light: 'gray.100',
+};
+
+export const secondaryHoverBgColor = {
+  dark: 'gray.700',
+  light: 'gray.200',
+};
+
+export default function NetworkListItem(props: UseRadioProps & { isLoading: boolean }) {
+  const { getCheckboxProps, getInputProps } = useRadio(props);
+  const { colorMode } = useColorMode();
+  const {
+    isChecked, isDisabled, isLoading, value,
+  } = props;
+  const input = getInputProps();
+  const checkbox = getCheckboxProps();
+  return (
+    <Box as="label">
+      <input disabled={isDisabled && (value === LOCAL_NODE_URL)} {...input} />
+      <Box
+        {...checkbox}
+        cursor="pointer"
+        borderRadius="md"
+        bgColor={secondaryBgColor[colorMode]}
+        _checked={{
+          bg: 'teal.600',
```

Comment:
For consistency, move these to a variable like the other colors (even if the color should be the same in light/dark mode).